bundles/org.openhab.ui/web/src/pages/developer/developer-tools.vue

```diff
@@ -27,6 +27,12 @@
                 <f7-list-item media-item title="Block Libraries" footer="Develop custom extensions for Blockly scripts" link="blocks/">
                   <f7-icon slot="media" f7="ticket" color="gray" />
                 </f7-list-item>
+                <f7-list-item media-item title="Copy DSL Definitions for All Things" footer="Copy all Things' DSL definitions to clipboard" link="#" @click="copyThingsDsl">
+                  <f7-icon slot="media" f7="lightbulb" color="gray" />
+                </f7-list-item>
+                <f7-list-item media-item title="Copy DSL Definitions for All Items" footer="Copy all Items' DSL definitions to clipboard" link="#" @click="copyItemsDsl">
+                  <f7-icon slot="media" f7="square_on_circle" color="gray" />
+                </f7-list-item>
                 <f7-list-item media-item title="Add Items from Textual Definition" footer="Create or update items &amp; links in bulk" link="add-items-dsl">
```

Comment:
Let's put the title inline with the other titles:


```suggestion
                <f7-list-item media-item title="Add Items from DSL Definition" footer="Create or update items &amp; links in bulk" link="add-items-dsl">
```

I will commit that.